.github/workflows/ci.yml

```diff
@@ -13,36 +13,38 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 9
       - uses: actions/setup-node@v2
         with:
           node-version: 18
-          cache: npm
+          cache: "pnpm"
       - name: Install
-        run: npm install
+        run: pnpm install
       - name: Build
-        run: npm run build
+        run: pnpm build
       - name: Run tests
-        run: npm run test
-      - name: Run tests in examples
-        run: npm run test:examples
```

Comment:
Is the intent that we won't run example tests at all now?

Is there anyone to keep the tests for CI time?

Reply:
My understanding is that we are not bringing the examples over to the Hardhat repo, so, since there were non-trivial `node_modules` resolution problems specific to the examples blocking my previous attempt at this PR, I decided it wasn't worth dedicating dev time to solving those issues since we won't have the examples available in that way soon anyways. 

There may be other ways we can bring them over to the Hardhat repo in a different form, but I would consider that to be a separate issue.

Reply:
I am going to miss them.

I pushed a commit that updates the publishing instructions to say:

1. install with verdaccio
2. install in each of the examples
3. run test

It doesn't solve the developer experience of hacking on changes in the examples, but at least we still check that the examples work before a release.